lib/ansiblelint/__main__.py

```diff
@@ -31,6 +31,25 @@
 import six
 from ansiblelint import RulesCollection
 from ansiblelint.version import __version__
+import yaml
+import os
+
+config = None
+
+if os.path.exists(".ansible-lint"):
```

Comment:
This should be in a function called by `main` called `read_config_file` or something so we don't need executable code in the body.

It's probably also worth making the filename a parameter, in case people want a system default or to pass in a configuration file name to override.

In order of precedence:
* A command line option (-F maybe)
* The cwd .ansible-lint file
* A file in the result of `AppDirs("ansible-lint", "com.github.willthames").user_config_dir` (this becomes ~/.config/ansible-review/ on linux)
* System file /etc/ansible-lint/ansible-lint.yml or something